src/transformers/models/rt_detr/__init__.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING
+
+from ...utils import OptionalDependencyNotAvailable, _LazyModule, is_torch_available, is_vision_available
+
+
+_import_structure = {"configuration_rt_detr": ["RT_DETR_PRETRAINED_CONFIG_ARCHIVE_MAP", "RTDetrConfig"]}
+
+try:
+    if not is_vision_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["image_processing_rt_detr"] = ["RTDetrImageProcessor"]
+
+try:
+    if not is_torch_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_rt_detr"] = [
+        "RTDetrForObjectDetection",
+        "RTDetrModel",
+        "RTDetrPreTrainedModel",
+    ]
+    _import_structure["modeling_resnet_rt_detr"] = [
+        "RTDetrTimmBackbone",
```

Comment:
I guess we can call this class `RTDetrResnetBackbone`?

Reply:
It is only supported trough TimmBackbone because the pretrained weight is converted from the TimmBackbone. Should I change this to ResnetBackbone?